docs/operations/figures/k8s-ces-overview-setup_de.puml

```diff
@@ -0,0 +1,85 @@
+@startuml
+!define CLOUDOGUURL https://raw.githubusercontent.com/cloudogu/plantuml-cloudogu-sprites/master
+
+!includeurl CLOUDOGUURL/common.puml
+!includeurl CLOUDOGUURL/dogus/cloudogu.puml
+!includeurl CLOUDOGUURL/tools/k8s.puml
+!define SECONDARY_COLOR #55EE55
+!define MANUAL_INSTALLATION #yellow
+!define MIXED_INSTALLATION #yellow|55EE55
+
+'cloud cloudogusvc as "Cloudogu services" <<$cloudogu>>
+cloud publicimgs as "public images\n(ohne Credentials)" #white
+
+rectangle "Cluster" as cluster <<$k8s>> {
+  rectangle "Namespace" as namespace <<$k8s>> #white {
+
+    rectangle credentials as " " #white {
+      file cert as "<<secret>>\neigenes oder \nself signed \nZertifikat" MIXED_INSTALLATION
+      file creds1 as "<<secret>>\nDogu registry credentials" MANUAL_INSTALLATION
+      file creds2 as "<<secret>>\nImage registry credentials" MANUAL_INSTALLATION
+    }
+
+    file dogujson as "<<configmap>>\nk8s-dogu-json" MANUAL_INSTALLATION
```

Comment:
Do you mean setup.json? There is a setup-config with the operator versions etc. as well.

Reply:
Hmm, this should be more refined, yes. 

- [ ] done